packages/raml-generator/src/main.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.apiSpecSourcePath);
+
+  // Check if the spec file exists.
+  fs.access(resolvedSourcePath, fs.constants.F_OK, err => {
```

Comment:
There is a simpler function to use: `fs.existsSync`

Reply:
I'm not sure what you mean - it's not "simpler" from my perspective.

Reply:
```js
if (!fs.existsSync(resolvedSourcePath)) {
  // do something
}
```